build-system/eslint-rules/no-import-rename.js

```diff
@@ -77,7 +75,9 @@ module.exports = function (context) {
 
     for (let i = 0; i < references.length; i++) {
       const ref = references[i];
-      const node = context.getNodeByRangeIndex(ref.identifier.start);
+      const [start] = ref.identifier.range;
+      const node = context.getNodeByRangeIndex(start);
+
```

Comment:
FIY this was failing CI after path update because of the same eslint breaking change from `node.start/end` to `node.range` as fixed in #37422.

I'm opening a separate PR to fix the remaining rules.